.github/workflows/weekly-build.yaml

```diff
@@ -9,43 +9,14 @@ env:
   GO_VERSION: "1.18"
   IMG: "quay.io/ocs-dev/lvm-operator:weekly"
 
+
```

Comment:
I think env from this flow can be removed

Reply:
Done.